### PR TITLE
docs: add table of network addresses data types

### PR DESCRIPTION
**Pull request by fzn0x**
Looks like missing syntax highlighting :)

### CODE REVIEW

**Comment by fzn0x** [comment]
Since the data types of the network addresses are different and not one type with different examples like Arrays and Range, maybe I will use a table instead.

**Review by ephys — APPROVED** [review]

**Comment by ephys** [comment]
Impacted page looks good: https://6330355bc9d6db658307c300--sequelize-site.netlify.app/docs/v6/other-topics/other-data-types/#network-addresses

**Comment by ephys** [comment]
Actually, the page must be renamed to `.mdx` to be able to use `DialectTableFilter`

**Review by ephys — CHANGES_REQUESTED** [review]
Rename to .mdx

**Comment by fzn0x** [comment]
Thanks! Done rename the file extension to `.mdx`

**Comment by ephys** [comment]
You'll need to update the files that link to this one, and I think you also need to import `DialectTableFilter` at the top of this file (see our other .mdx to find out how to do that)

**Comment by fzn0x** [comment]
Thanks, learn a new thing! `mdx` was awesome

File: versioned_docs/version-6.x.x/other-topics/other-data-types.mdx [hunk 1]
```diff
@@ -1,7 +1,11 @@
 ---
+sidebar_position: 2
 title: Other Data Types
 ---
 
+import { DialectTableFilter } from '@site/src/components/dialect-table-filter.tsx';
+import { Deprecated } from '@site/src/components/deprecated.tsx';
```
Comment (ephys) [inline versioned_docs/version-6.x.x/other-topics/other-data-types.mdx:7]: deprecated is not used

**Review by ephys — APPROVED** [review]

**Pull request merged**